repo/packages/B/beta-tensorflow/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "packagingVersion": "4.0",
+    "minDcosReleaseVersion": "1.9",
+    "upgradesFrom": [
+    ],
+    "downgradesTo": [
+    ],
```

Comment:
Can we keep the upgradesFrom/downgradesTo value to be the current value - "2.0.2-1.3.0-beta"?

Reply:
This is the first release, so I don't think there's anything to upgrade or downgrade to right?

Reply:
@elezar Do we want to release this as "beta-tensorflow" or do we want it to be "tensorflow"?